pandas/core/series.py

```diff
@@ -1062,7 +1063,24 @@ def _get_value(self, label, takeable: bool = False):
 
         # Similar to Index.get_value, but we do not fall back to positional
         loc = self.index.get_loc(label)
-        return self.index._get_values_for_loc(self, loc, label)
+
+        if is_integer(loc):
```

Comment:
So since this PR has been opened code is no longer moved from `Index._get_values_for_loc`?

Reply:
get_values_for_loc can be removed after this.  at the time this PR was opened there was another (deprecated) usage